examples/webhook-signing/nextjs/api-router/route.ts

```diff
@@ -0,0 +1,43 @@
+import { NextRequest, NextResponse } from 'next/server';
```

Comment:
Can you rename the directory so it resembles the Next.js project structure (`app/api/stripe/webhook/route.ts`)?